app/src/org/commcare/activities/connect/ConnectIdDatabaseHelper.java

```diff
@@ -136,4 +140,74 @@ public static void setRegistrationPhase(Context context, ConnectIdTask phase) {
             storeUser(context, user);
         }
     }
+
+    public static void storeAvailableJobs(Context context, List<ConnectJob> jobs) {
```

Comment:
Agreed. I'll also rename `ConnectIdNetworkHelper` to drop the Id since it's also being used more widely now. Is there a better place for these classes to live instead of in the `activities` folder?

Reply:
Adding ConnectIdSsoHelper to this list also